website/docs/Examples/AutoML-NLP.md

```diff
@@ -9,25 +9,35 @@ pip install "flaml[nlp]"
 
 ### A simple sequence classification example
 
+An example of sequence classification data is the Stanford Sentiment Treebank data (SST) data:
+
+sentence  label 
+hide new secretions from the parental units       0 
+contains no wit , only labored gags       0 
```

Comment:
include one positive example.
use markdown to format it so that it's clear to see it's a table.